app/models/interview.rb

```diff
@@ -8,4 +8,24 @@ class Interview < ActiveRecord::Base
   validates :lastname, :firstname, presence: true
   validates :template_id, presence: true
   validates :target_level, presence: true, inclusion: { in: TARGET_LEVELS, message: "%{value} is not a valid level" }
+
+  def self.not_answered_first_question(interview)
+    mark_nil_first = 1
+    mark_nil = false
+    interview.interview_questions.each do |q|
+      if q.mark.nil? && mark_nil == false
+        mark_nil_first = q.id
+        mark_nil = true
+      end
+    end
+    mark_nil_first
+  end
+
+  def self.not_answered_questions(interview)
```

Comment:
instance method
good practice is to avoid using ruby if it is possible to accomplish task using sql, for example this one should be solved:
```ruby
def not_answered_questions
  interview_questions.where('`mark` IS NULL')
end
```